optimum/intel/__init__.py

```diff
@@ -55,6 +62,15 @@
         "IPEXModel",
     ]
 
+try:
+    if not (is_ipex_available() and is_sentence_transformers_available()):
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    _import_structure["utils.dummy_ipex_objects"].extend(["IPEXSentenceTransformer"])
```

Comment:
it broke OpenVINO path 
https://github.com/openvinotoolkit/openvino/actions/runs/12240792041/job/34146426674